lib/sift/where_handler.rb

```diff
@@ -5,7 +5,28 @@ def initialize(param)
     end
 
     def call(collection, value, _params, _scope_params)
-      collection.where(@param.internal_name => value)
+      if @param.type == :jsonb
+        apply_jsonb_conditions(collection, value)
+      else
+        collection.where(@param.internal_name => value)
+      end
+    end
+
+    private
+
+    def apply_jsonb_conditions(collection, value)
+      return collection.where("#{@param.internal_name} @> ?", val.to_s) if value.is_a?(Array)
+
+      value.each do |key, val|
+        condition = if val.is_a?(Array)
+          "#{@param.internal_name} @> '{\"#{key}\": [?]}'"
+        else # Single Value
+          val = val.to_s
+          "#{@param.internal_name}->>'#{key}' = ?"
+        end
+        collection = collection.where(condition, val)
```

Comment:
If I were to filter multi-select, will this filter by OR or AND ?

Reply:
I just updated this PR, now it works with an OR strategy for multi-select, and for single-select an array can be sent as a parameter and it will return all records with any of the possible values.

Reply:
README has been updated as well:
https://github.com/moisesnarvaez/sift/tree/mn/add_jsonb#filter-on-jsonb-column